example/ios/AirMapsExplorer/AppDelegate.m

```diff
@@ -11,12 +11,16 @@
 
 #import "RCTRootView.h"
 
+#import <GoogleMaps/GoogleMaps.h>
+
 @implementation AppDelegate
 
 - (BOOL)application:(UIApplication *)application didFinishLaunchingWithOptions:(NSDictionary *)launchOptions
 {
   NSURL *jsCodeLocation;
 
+  [GMSServices provideAPIKey:@"AIzaSyAeHIC4IG7XKT2Ls5Ti_YZV-6DHQk6dVHE"];
```

Comment:
Let's use the same key we're using for the Android example app:

https://github.com/airbnb/react-native-maps/blob/2b88e76c7bf2fc3112a892ecc86bec2d8ca63560/example/android/app/src/main/AndroidManifest.xml#L24